### ISSUE TITLE
Bug with filemtime() for static elements in Windows

### ISSUE THREAD

**Issue by sergant210** [modxcms/revolution#15304]
### Summary
If you have static elements (snippets, plugins, chunks) some times you'll get this error (in Windows).
```
[2020-10-18 23:33:54] (ERROR @ D:\Projects\test.local\www\core\model\modx\modscript.class.php : 146) PHP warning: filemtime(): stat failed for D:/Projects/test.local/www/core/cache/includes/elements/modplugin/26.include.cache.php
```
[modScript::loadScript()](https://github.com/modxcms/revolution/blob/bcb4652ee261e82c0771f2322b0f91b9ac8f9153/core/model/modx/modscript.class.php#L140)
![modx static](https://user-images.githubusercontent.com/4679725/96412551-02679500-11f3-11eb-8105-8d09bd070f1f.png)

### Step to reproduce
1. Create a static snippet or plugin.
2. Clear the cache.
3. Refresh the page.
4. Open the error log.

### Environment
MODX 2.8.0. 
PHP 7.4. 
Windows 7, 10.

**Comment by opengeek** [comment]
The `filemtime()` check will not be executed if the `$includeFilename` is not readable, so your screenshot is not identifying the correct issue.

**Comment by sergant210** [comment]
You are wrong. Take a look at the [documentation](https://www.php.net/manual/en/function.filemtime.php).

**Comment by opengeek** [comment]
> You are wrong. Take a look at the [documentation](https://www.php.net/manual/en/function.filemtime.php).

I don't know what you are talking about. That call never executes if `$result` is `false`which it will always be if the file does not exist yet. Take a look at the code.

**Comment by sergant210** [comment]
I see. I was inattentive. Then why does this error appear? The problem looks like the [lexicon cache issue](https://github.com/modxcms/revolution/issues/14309).

**Comment by opengeek** [comment]
> I see. I was inattentive. Then why does this error appear? The problem looks like the [lexicon cache issue](https://github.com/modxcms/revolution/issues/14309).

I'm having no luck reproducing the issue, so I do not know yet.

**Comment by sergant210** [comment]
![static](https://user-images.githubusercontent.com/4679725/96480793-6023cd80-1243-11eb-9ceb-4334bd5862c1.gif)

**Comment by opengeek** [comment]
I'm afraid this must be something unique to that particular plugin. I cannot recreate the issue with a basic plugin.

**Comment by sergant210** [comment]
It's strange. I got this issue in my laptop with several static plugins. (The picture above from my desktop)

Ok. I'll dig further.

**Comment by sergant210** [comment]
I think I got it. This happens only on Windows.

**Comment by Mark-H** [comment]
Anything the core should take into consideration?

**Comment by sergant210** [comment]
@Mark-H I have no idea. There are no problems on Ubuntu.

**Comment by Ruslan-Aleev** [comment]
@Mark-H @sergant210 Windows has a problem with filemtime, maybe this comment will come in handy https://www.php.net/manual/en/function.filemtime.php#100692

**Comment by Ruslan-Aleev** [comment]
Maybe we should reopen this issue? Because the error log is constantly full of unnecessary lines.

**Comment by elmys** [comment]
Yes, it is actual still on Windows 10 + Docker.
